diff --git a/dist/runtime/plugins/view/client.js b/dist/runtime/plugins/view/client.js
index efbaf461f2d557002c3dd3f1cc5b18a765d3ff6e..4f12fe8cc5f6562a87685ad21e8fb7153252950a 100644
--- a/dist/runtime/plugins/view/client.js
+++ b/dist/runtime/plugins/view/client.js
@@ -5,6 +5,8 @@ import { debounce } from "perfect-debounce";
 import { events as inspectorEvents, hasData as inspectorHasData, state as inspectorState } from "vite-plugin-vue-tracer/client/overlay";
 import { computed, createApp, h, markRaw, ref, shallowReactive, shallowRef, toRef, watch } from "vue";
 import { initTimelineMetrics } from "../../function-metrics-helpers.js";
+import * as utils from 'vite-plugin-vue-tracer/client/record'
+
 import Main from "./Main.vue";
 import { popupWindow, state } from "./state.js";
 const clientRef = shallowRef();
@@ -173,7 +175,9 @@ export async function setupDevToolsClient({
       },
       toggle: () => {
         inspectorState.isEnabled = !inspectorState.isEnabled;
-      }
+      },
+      getInspectorState: () => inspectorState,
+      utils
     });
   }
   setupRouteTracking(timeline, router);
